include/rang.hpp

```diff
@@ -261,28 +273,29 @@ void init()
 	rang_implementation::RANG_coutbuf();
 	rang_implementation::RANG_cerrbuf();
 	rang_implementation::RANG_clogbuf();
+    rang_implementation::isColorSupported() =
+        rang_implementation::supportsColor();
 }
 
 template <typename T>
 inline rang_implementation::enableStd<T> operator<<(
   std::ostream &os, T const value)
 {
 	std::streambuf const *osbuf = os.rdbuf();
-	return (os.iword(rang_implementation::getIword())
-	         || ((rang_implementation::supportsColor())
-	         && (rang_implementation::isTerminal(osbuf))))
-	  ? rang_implementation::setColor(os, value)
-	  : os;
+	return rang_implementation::isColorForced() || (
+        rang_implementation::isColorSupported() &&
+	    rang_implementation::isTerminal(osbuf)
+    ) ? rang_implementation::setColor(os, value) : os;
 }
 
 template <typename T>
 inline rang_implementation::enableControl<T> operator<<(
   std::ostream &os, T const value)
 {
 	if (value == rang::control::forceColor) {
-		os.iword(rang_implementation::getIword()) = 1;
+        rang_implementation::isColorForced() = true;
```

Comment:
for about 2 minutes I was thinking why compiler isn't giving any errors about this line. That was before I saw the reference as return value 👍